virttest/remote.py

```diff
@@ -530,7 +531,7 @@ def scp_to_remote(host, port, username, password, local_path, remote_path,
                "-o StrictHostKeyChecking=no "
                "-o PreferredAuthentications=password -r %s "
                "-P %s %s %s@\[%s\]:%s" %
-               (limit, port, local_path, username, host, remote_path))
+               (limit, port, local_path, username, pipes.quote(host), pipes.quote(remote_path)))
```

Comment:
Shouldn't you quote local_path too?

Reply:
Oups, I accidentally quote host instead of local_path. Updating...